### PR TITLE
Added dockerlinter parser analysis

### PR DESCRIPTION
**Pull request by iamabhishek-dubey**
Signed-off-by: iamabhishek-dubey <abhishekbhardwaj510@gmail.com>

### CODE REVIEW

**Comment by iamabhishek-dubey** [comment]
This Jenkins build is not failed because of my code changes, Please review :D

**Comment by iamabhishek-dubey** [comment]
I will fix this codecov alerts

**Comment by iamabhishek-dubey** [comment]
Fixed codecov warning, the job is failed because of the JNLP connection issue

File: src/test/java/edu/hm/hafner/analysis/parser/OTDockerLintParserTest.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    @Test
+    void accepts() {
+        assertThat(new OTDockerLintParser().accepts(
+                new FileReaderFactory(FileSystems.getDefault().getPath("lint.json")))).isTrue();
+        assertThat(new OTDockerLintParser().accepts(
+                new FileReaderFactory(FileSystems.getDefault().getPath("foo.txt")))).isFalse();
+    }
+
+    @Test
+    void unusualInput() {
```
Comment (uhafner) [inline src/test/java/edu/hm/hafner/analysis/parser/OTDockerLintParserTest.java:60]: What exactly is the difference to the other test (ot-docker-linter.json)?

**Review by uhafner — CHANGES_REQUESTED** [review]

**Comment by iamabhishek-dubey** [comment]
Please review @uhafner

**Review by uhafner — APPROVED** [review]

**Comment by uhafner** [comment]
Thanks!

**Pull request merged**